pandas/tests/indexes/numeric/test_numeric.py

```diff
@@ -50,19 +43,17 @@ def float_index(self, dtype):
         return self._index_cls([0.0, 2.5, 5.0, 7.5, 10.0], dtype=dtype)
 
     def test_repr_roundtrip(self, index):
-        tm.assert_index_equal(eval(repr(index)), index, exact=True)
+        from pandas.core.api import NumericIndex  # noqa: F401
```

Comment:
What's this used for?

Reply:
This is just temporary crutch until we remove `NumericIndex`. So it will be gone again in a few PRs.

The issue is that `repr(index)` gives e.g. `"NumericIndex([1, 2, 3], dtype='int32')"`. To call `eval` on this, we need `NumericIndex` in the local namespace.

Reply:
Got you, thx